### DISCUSSION TITLE
Behavior update of the range dates selection

### DISCUSSION THREAD

**Discussion by GilbertHG** [easepick/easepick#211] [Ideas]
Hi, I think it's better if you update the behavior of backward dates selection, if user do backward dates selection instead of reverse the start and end date, you could repeat the date selection and make the last date picked as start date. Example:

![ezgif com-video-to-gif](https://github.com/easepick/easepick/assets/50436511/56da9923-442c-40b7-87dc-089c8ec87621)

**Comment by cmubo** [comment]
Are you looking for the [repick](https://easepick.com/packages/range-plugin.html#option-repick) option?
